samples/src/main/res/layout/activity_style_sample.xml

```diff
@@ -57,7 +57,9 @@
             app:fabMenu="@menu/menu_main"
             app:miniFabBackgroundTint="@android:color/white"
             app:miniFabDrawableTint="?attr/colorPrimaryDark"
-            app:miniFabTitleTextColor="?attr/colorPrimaryDark" />
```

Comment:
Leave this layout as it is and add a new activity to show how to use custom layouts for the menu items.